tests/python/rest_api/test_analytics.py

```diff
@@ -35,11 +35,17 @@ def _test_cannot_see(self, user):
         assert response.status_code == HTTPStatus.FORBIDDEN
 
     @pytest.mark.parametrize(
-        "privilege, is_allow",
-        [("admin", True), ("business", True), ("worker", False), ("user", False)],
+        "conditions, is_allow",
+        [
+            (dict(privilege="admin"), True),
+            (dict(privilege="business"), True),
```

Comment:
Should you update `has_analytics_access` in the test db/assets for users with `business` privilege too?

Reply:
Or sorry,  it is my bad. Probably it will be better to add a comment that users with business privilege should have analytics access only when the cvat server is configured with `RESTRICTIONS['analytics_visibility'] = True`. Perhaps we should test that too when there will be a pipeline for testing different server configurations.

Reply:
I also see a problem that existed before and has not been fixed here:
If the server is configured with `RESTRICTIONS['analytics_visibility'] = True` then a user with the `business` group will not see the analytics tab on the CVAT UI.  Probably need to think about it or discuss this moment one more time with the team. @azhavoro, Could you please remind me why it was implemented so and do we still need this server configuration or can it be replaced with a new approach(with `has_analytics_access`)?